crypto/init.c

```diff
@@ -560,6 +572,11 @@ int OPENSSL_init_crypto(uint64_t opts, const OPENSSL_INIT_SETTINGS *settings)
     if ((opts & OPENSSL_INIT_ENGINE_OPENSSL)
             && !RUN_ONCE(&engine_openssl, ossl_init_engine_openssl))
         return 0;
+# if !defined(OPENSSL_NO_HW) && !defined(OPENSSL_NO_DEVCRYPTOENG)
```

Comment:
Actually, no.  There's no hardware to disable, at least from our perspective, so `no-hw` shouldn't be considered at all here.
(as a matter of fact, `no-hw` should pretty much be a no-op for the moment)